### PR TITLE
Adding Travis CI checking and deployment to new repo

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

**Comment by ryelo** [comment]
This is failing due to indentation spacing. Updating that too.

File: .eslintrc.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ rules:
     - unix
   max-len:
     - 2
-    - 130
+    - 1000
```
Comment (Kinlaw) [inline .eslintrc.yml]: max-len 1000?

**Comment by ryelo** [comment]
@Kinlaw whoops. I was debugging. It's at 150 now.

**Comment by Kinlaw** [comment]
Looks Good! I have no write permissions :(

**Pull request merged**